### DISCUSSION TITLE
Контактные данные пользователей

### DISCUSSION THREAD

**Discussion by gfg7** [habralab/garnet-team#143] [Идеи]
Добавить в сущность пользователя поле, где можно указать контактные данные - номер телефона/почту/ссылку на свой аккаунт в соц-сетях или личный сайт и т.д. в неограниченном виде
+ Для чего
Владельцам и пользователям будет проще связаться и переговорить вступление/приглашение в команду/проект и сделать решение
Пользователи смогут оставить свой контакт, если ими заинтересуются для участия в коммерческом проекте, посмотрев их результаты работы на нашем ресурсе

+ Почему нельзя оставить это в описании
Будет мешать полнотекстовому поиску, нерелевантные результаты

+ Нужно дорабатывать поиск? Нет
Никто не будет искать пользователей по его контактам

Доработка будет состоять только в его показе в карточке пользователя и редактировании этого параметра

**Comment by Demorey** [comment]
> Добавить в сущность пользователя поле, где можно указать контактные данные - номер телефона/почту/ссылку на свой аккаунт в соц-сетях или личный сайт и т.д. в неограниченном виде
> 
> * Для чего
>   Владельцам и пользователям будет проще связаться и переговорить вступление/приглашение в команду/проект и сделать решение

Вообще я бы даже на данном этапе сделал что хотя бы одно поле из контактных данных должно быть заполнено обязательно, потому как сейчас у нас в проекте нет никакой прямой коммуникации между пользователями, и если у пользователя не будет указан хотя бы один контакт для связи, ему банально все будут отказывать во вступлении в команду/проект.
